### ISSUE TITLE
A11y - check slider module and improve

### ISSUE THREAD

**Issue by anqaka** [SnowdogApps/magento2-alpaca-components#374]
To check:

- Focus management
- Keyboard support
- Screen reader
- aria attributes
- a11y tree

**Comment by anqaka** [comment]
In components looks good. It should be checked in theme for product slider as related.
